app/abilities/sbv/group_ability.rb

```diff
@@ -22,18 +23,29 @@ module Sbv::GroupAbility
       permission(:uv_lohnsumme).may(:show_uv_lohnsummen).everywhere_if_admin
       permission(:uv_lohnsumme).may(:edit_uv_lohnsummen).everywhere_if_admin_or_in_same_layer
     end
+  end
 
-    def everywhere_if_admin
-      [
-        Group::Generalverband::Admin,
-        Group::Root::Admin,
-        Group::Mitgliederverband::Admin,
-        # Group::Regionalverband::Admin # is not allowed, according to hitobito/hitobito_sbv#30
-      ].any? { |admin_role| role_type?(admin_role) }
+  def if_writable
+    [
+      :layer_and_below_full,
+      :layer_full,
+      :group_and_below_full,
+      :group_full
+    ].any? do |permission|
```

Comment:
Ist das wirklich die beste Art das zu lösen? Weshalb geht nicht etwas ähnliches wie:
```
can? :manage, group
```

Reply:
Im Kontext einer einzelnen Ability habe ich keine entsprechende Abkürzung gefunden.

Im controller/view-Bereich könnte man das so lösen, müsste aber jeden Aufruf von `can :deleted_groups, group` entsprechend ergänzen (natürlich ohne Garantie, dass es auch in Zukunft geht). 

Ich fand das anpassen der zugrundeliegenden Ability etwas vollständiger. Der Preis, die einzelen permissions zu prüfen war für mich okay. Übersehe ich hier eine schöne high-level-methode, um das lesbarer/wartbarer zu gestaltet?